android/src/main/java/com/appgoalz/rnjwplayer/CastOptionsProvider.java

```diff
@@ -45,6 +48,11 @@ public CastOptions getCastOptions(Context context) {
                 .setReceiverApplicationId(DEFAULT_APPLICATION_ID)
                 .setCastMediaOptions(mediaOptions)
                 .setLaunchOptions(launchOptions)
+
```

Comment:
I believe these settings were added for testing purposes. the ones which are not required can be removed.